bin/physlr-make

```diff
@@ -1021,6 +1021,10 @@ min_path_size=200
 %.map.$(ref).n10.paf.gz: %.path $(lr).k$k-w$w.n$(minimum_barcode_multiplicity)-$(maximum_barcode_multiplicity).c2-x.physlr.tsv $(name)/$(ref).k$k-w$w.physlr.tsv
 	$(time) $(python) $(bin)/physlr map-paf -V$V -n10 $^ | $(gzip) >$@
 
+# Map the reference to the backbone graph generated with split minimizers and output PAF.
```

Comment:
```suggestion
# Map the reference to the backbone graph with split minimizers and output PAF.
```
Is this correct? We're mapping with split minimizers, but the backbone graph isn't generated with split minimizers?

Reply:
Yeah your change is correct.